src/Validation/Validation.csproj

```diff
@@ -56,6 +59,29 @@
     <PackageReference Include="MSBuild.Sdk.Extras" Version="1.1.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.1'">
+    <PackageReference Include="Nullable" Version="1.3.0" />
```

Comment:
I had it that way at first and wasn't a huge fan of the fact that it still showed as a library when you view the dependencies for .NET Standard 2.1. I thought it might end up being confusing in the long-run, but have no problem dropping the condition. I didn't even think about the fact that it should have had `PrivateAssets="all"` on it though that's easy enough to add.